genon/preprocessor/facade/oneagent_processor.py

```
@@ -0,0 +1,1646 @@
+        include_wmf = kwargs.get('include_wmf', False)
+
+        # save_images 옵션이 현재 설정과 다르면 컨버터 재생성
+        if (self.simple_pipeline_options.save_images != save_images or
+            getattr(self.simple_pipeline_options, 'include_wmf', False) != include_wmf):
+            self.simple_pipeline_options.save_images = save_images
+            self.simple_pipeline_options.include_wmf = include_wmf
+            self._create_converters()
+
+        try:
+            conv_result: ConversionResult = self.ocr_converter.convert(file_path, raises_on_error=True)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)